src/Core/View.js

```diff
@@ -87,6 +87,18 @@ function View(crs, viewerDiv, options = {}) {
     }
 
     this._delayedFrameRequesterRemoval = [];
+
+    this._allLayersAreReadyCallback = () => {
+        // all layers must be ready
+        const reducer = (allReady, layer) => allReady && layer.ready;
+        const allReady = this.getLayers().reduce(reducer, true);
```

Comment:
We have a shortcut for this: [Array.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every).

(note that we still don't have a `Array.remove(object)`, but okay, javascript ;-) )

Reply:
Done